schemas/HPXMLBaseElements.xsd

```diff
@@ -6611,8 +6611,84 @@
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			<xs:element minOccurs="0" name="ServiceFeeders">
+				<xs:annotation>
+					<xs:documentation>Records the electrical loads of major electric and other specific systems for sizing the service entrance conductors (wires going into the main electrical panel from the utility pole) or the feeder conductors (wires between electrical panels) according to the National Electrical Code. This calculation is known as the service load or feeder load calculation. The electrical load refers to the nominal or nameplate power rating of a device in Volt-Amps or Watts. It can be calculated as the product of voltage and full load amp (FLA) or rated load amp (RLA). Minimum circuit amp (MCA) or maximum overcurrent rating can be used but are more conservative than FLA or RLA.</xs:documentation>
+				</xs:annotation>
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element maxOccurs="unbounded" minOccurs="1" name="ServiceFeeder" type="ServiceFeederType"/>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
 			<xs:element minOccurs="0" ref="extension"/>
 		</xs:sequence>
 		<xs:attribute name="dataSource" type="DataSource"/>
 	</xs:complexType>
+	<xs:complexType name="BranchCircuitType">
+		<xs:sequence>
+			<xs:group ref="SystemInfo"/>
+			<xs:element minOccurs="0" name="Voltage" type="ElectricPanelVoltage"/>
+			<xs:element minOccurs="0" name="MaxCurrentRating" type="Current">
+				<xs:annotation>
+					<xs:documentation>[A]</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="OccupiedSpaces" type="BranchCircuitOccupiedSpaces">
+				<xs:annotation>
+					<xs:documentation>Number of occupied breaker spaces.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element minOccurs="0" name="AttachedToComponent" type="LocalReference" maxOccurs="unbounded"/>
+			<xs:element minOccurs="0" name="AttachedToSubPanel" type="LocalReference">
+				<xs:annotation>
+					<xs:documentation>References the ElectricPanel that this BranchCircuit is attached to.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+		<xs:attribute name="dataSource" type="DataSource"/>
+	</xs:complexType>
+	<xs:complexType name="ServiceFeederType">
+		<xs:sequence>
+			<xs:group ref="SystemInfo"/>
+			<xs:element minOccurs="0" name="LoadType" type="ServiceFeederLoadType"/>
+			<xs:element minOccurs="0" name="PowerRating" type="HPXMLDouble">
+				<xs:annotation>
+					<xs:documentation>Volt-Amps [VA] or Watts [W]</xs:documentation>
```

Comment:
We should just specify one unit here. There's [a difference between VA and W](https://www.eaton.com/de/en-gb/products/backup-power-ups-surge-it-power-distribution/backup-power-ups/va-versus-watts--eaton.html). Which is most important for this use case?

Reply:
Just a thought... support both? Similar to, e.g., AnnualHeatingEfficiency for HeatingSystem:

```
<AnnualHeatingEfficiency>
  <Units>AFUE</Units>
  <Value>0.92</Value>
```
```
<PowerRating>
  <Units>VA or W</Units>
  <Value>0.92</Value>
```

Reply:
If you're going to allow both in the schema, then you would need to update OS-HPXML to handle both. It seems like you currently assume it's W? https://github.com/NREL/OpenStudio-HPXML/blob/electric_panel/docs/source/workflow_inputs.rst?plain=1#L4700

Reply:
I would? Couldn't I just exclude VA using the EPvalidator?

Reply:
For load calculation, it's both and but it's used interchangeably there and do not distinguish between real and reactive portion of the power. If we can't support both, then just go with W.

Reply:
Let's just go with W if for the most common use case they're not differentiating.

Reply:
Done in [d347083](https://github.com/hpxmlwg/hpxml/pull/435/commits/d34708353c555ab33015296db5c59dc8a6fc182b).